bundles/io/org.eclipse.smarthome.io.rest.core/OSGI-INF/configurableserviceresource.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" name="org.eclipse.smarthome.io.rest.core.service.ConfigurableServiceResource">
+   <implementation class="org.eclipse.smarthome.io.rest.core.service.ConfigurableServiceResource"/>
+   <service>
+      <provide interface="org.eclipse.smarthome.io.rest.RESTResource"/>
+      <provide interface="org.eclipse.smarthome.io.rest.core.service.ConfigurableServiceResource"/>

```

Comment:
All valid remarks, but I wonder why we at all have to export this class in the first place? Isn't it enough to register it as a RESTResource service? I see that the others are also providing RESTResource and their class, but imho this could be changed as well.